image_classification/android/shell/cxx/image_classification/run.sh

```diff
@@ -1,9 +1,35 @@
 #!/bin/bash
+MODEL_NAME=PP-LCNet_x1_0
+MODEL_LIST="PP-LCNet_x1_0 MobileNetV3_small_x1_0"
+
+if [ -n "$1" ]; then
+  MODEL_NAME="$1"
+fi
+
+if ! echo "$MODEL_LIST" | grep -qw "$MODEL_NAME"; then
+  echo "Supported model: ${MODEL_LIST}"
+  echo "$MODEL_NAME is not in the support list. Exiting."
```

Comment:
`is not in the supported models. Now exiting.`

Reply:
done